### PR TITLE
WIP: Rename search to admin

### LINKED ISSUE #805: Rename the search app to admin
The search app won't be just search forever, it will eventually be the entire admin. The AMO pages will also get search so it will just be confusing.

### PR DESCRIPTION
**Pull request by tofumatt**
I'm not sure if this is the right approach and I'm still learning some react/redux stuff so this was done as much for a learning exercise as anything else.

Since I'm working on search I figured I'd give this a shot so I could make a real "search" 😄 

Fixes #805.

r?

### CODE REVIEW

File: docs/adding-a-page.md [hunk 1]
```diff
@@ -78,14 +78,14 @@ To render this component we'll tell [react-router](https://github.com/reactjs/re
...
 
 ```jsx
-// src/search/routes.js
+// src/admin/routes.js
 // ... omit imports
 
 export default (
   <Route path="/" component={App}>
     <Route component={LoginRequired}>
-      <Route path="search">
```
Comment (muffinresearch) [inline docs/adding-a-page.md]: The url here should still be 'search' shouldn't it?

**Comment by muffinresearch** [comment]
I think this has gone a bit too far, the app is now admin but the views and the urls etc that are for search should still be called search.

**Comment by muffinresearch** [comment]
You'll need to let ops know that the app target is changing so they can update their deployment scripts etc.

**Comment by tofumatt** [comment]
We chatted about this on vidyo and I'm gonna tweak this approach. Might just make a new PR, but don't r? for now 😄

**Pull request closed without merging**